### ISSUE TITLE
[BUG-REPORT] 'str' has no attribute '_fields' in from_json()

### ISSUE THREAD

**Issue by samayer12** [vaexio/vaex#1258]
**Description**

`vaex.from_json(path, orient='index')` returns the error `AttributeError: 'str' object has no attribute '_fields'` when called with `test_data.json.txt`. `.txt` appended to filename so I can upload it here.

Here's the format of one entry on `test_data.json`

    "YYY-MM-DD HH:MM:SS": {
        "4. close": "4.000000",
        "1. open": "5.000000",
        "2. high": "2.000000",
        "3. low": "3.000000",
        "5. volume": "5.000000"
    }

**Software information**
 - Vaex version info:

 ```
'vaex': '4.0.0',
'vaex-core': '4.0.1',
'vaex-viz': '0.5.0',
'vaex-hdf5': '0.7.0',
'vaex-server': '0.4.0',
'vaex-astro': '0.8.0',
'vaex-ui': '0.3.0',
'vaex-jupyter': '0.6.0',
'vaex-ml': '0.11.0',
'vaex-arrow': '0.5.1'
```

 - Vaex was installed via: `pip`
 - OS: `5.10.19-1-MANJARO`

**Additional information**
- Full stack trace: [vaex_error.txt](https://github.com/vaexio/vaex/files/6137897/vaex_error.txt)
- Test Dataset: [test_data.json.txt](https://github.com/vaexio/vaex/files/6137900/test_data.json.txt)

* Output from `pandas.read_json(path)`:

```
           2020-01-02 06:00:00  2020-01-02 06:01:00  2020-01-02 06:02:00
4. close                 87.01                86.81                 86.7
1. open                  87.15                86.85                 86.7
2. high                  87.15                86.85                 86.7
3. low                   87.01                86.81                 86.7
5. volume               539.00               250.00                120.0
```

* Output from `vaex.from_json(path, orient='index')`:
```
[...See Attached Stack Trace...]
  File "/usr/lib/python3.9/ast.py", line 249, in iter_fields
    for field in node._fields:
AttributeError: 'str' object has no attribute '_fields'
```

**Comment by samayer12** [comment]
If I use `vaex.from_csv(path)` with [test_data.csv.txt](https://github.com/vaexio/vaex/files/6137925/test_data.csv.txt), then I get the output below.

```
  #  time                     close    open    high    low    volume
  0  '2020-01-02 06:00:00'    87.01   87.15   87.15  87.01       539
  1  '2020-01-02 06:01:00'    86.81   86.85   86.85  86.81       250
  2  '2020-01-02 06:02:00'    86.7    86.7    86.7   86.7        120
```

This is usable... but I'll have to pre-process a lot of `.json` files.

**Comment by JovanVeljanoski** [comment]
Hi, 

Thanks for submitting this issue and for the means to reproduce your problem. 
I am afraid I can not reproduce this - however I am using python 3.7 & 3.8 (tried vaex v4 and v3 as well). 

Vaex is currently not tested on python 3.9 - it is something we hope to support properly soon, so the issue you are seeing might be connected to that.  Is it possible to try it out with python 3.7 or 3.8?

Also,  from your `vaex.__version__` i see you have the latest one installed, but for some reason the `vaex-arrow` package is still showing. That is now deprecated and part of `vaex-core`. Don't know if that has any influence, maybe it is just an artifact.

**Comment by samayer12** [comment]
Oh yeah, I'm on 3.9 so that's probably the source of the issue. I think I've seen some other libraries break in similar ways as well.

I'll dig into it a little more and see what's up.

**Comment by Eisbrenner** [comment]
Had the same issue with a CSV file on python 3.9; downgrading to python 3.8 fixed the issue.

**Comment by samayer12** [comment]
Yup, the issue was use in `3.9`. I'm using `asdf` ([link](https://stackoverflow.com/questions/2547554/multiple-python-versions-on-the-same-machine) to toggle between versions as needed. Thanks for the clarification!

**Comment by maartenbreddels** [comment]
Hi Sam,

thanks for the report indeed. We just fixed 3.9 support in https://github.com/vaexio/vaex/pull/1069 I hope to do a release soon, including a binary wheel for the major platforms.

cheers,

Maarten
